README.md

```diff
@@ -59,13 +59,21 @@ include snmp
 
 #### Deprecation Warning
 
-The classes `snmp::server` and `snmp::trapd` will be merged into class `snmp` in version 3.0.0 of this module.  All of their class parameters will be made available in the `snmp` class.
+##### Past module 3.x serie
 
-The parameter `install_client` has been renamed to `manage_client` in version 4.0.0 of this module.
+ * The classes `snmp::server` and `snmp::trapd` has been merged into class `snmp`.  All of their class parameters available in the `snmp` class.
```

Comment:
I think this should be “have been merged” because there are multiple classes. Also “the class”.